src/pages/GeneratingPage.tsx

```diff
@@ -30,11 +35,21 @@ export const GeneratingPage = () => {
             name="classToGenerate"
             ref={register}
           >
-            {mapperClassList.map((dndClass, index) => (
-              <option key={index} className="text-capitalize" value={dndClass}>
-                {dndClass === "class" ? "random" : dndClass}
-              </option>
-            ))}
+            {classes != null
+                  ? <Fragment>
+                    <option key={"dndClass" + 0} className="text-capitalize" value={'class'}>
+                      {"Random"}
+                    </option>
+                    {classes && classes.map((dndClass: any, index: number) => (
+                      <option key={"dndClass" + (index + 1).toString()} className="text-capitalize" value={dndClass.index}>
+                        {dndClass.name}
+                      </option>
+                    ))}
+                  </Fragment>
+                  : <option key={'loading'} className="text-capitalize" value={'loading'}>
```

Comment:
`key` would be unnecessary since it's not inside a for-loop and you can use `value="loading"` without the `{ }`